src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -482,15 +486,24 @@ export default class GroupComparisonStore extends ComparisonStore {
     }
 
     @computed get activeMutationMapperGene() {
-        let gene =
-            this.genes.result!.find(
-                g => g.hugoGeneSymbol === this.userSelectedMutationMapperGene
-            ) ||
-            this.genes.result!.find(
-                g =>
-                    g.hugoGeneSymbol ===
-                    this.genesSortedByMutationFrequency.result![0]
-            );
+        let gene;
+        if (this.genes.isComplete) {
```

Comment:
we should be checking in the view to make sure that gense is complete before we render any component which would be dependent on it.   this is an anti-pattern that we have a lot in the app unfortunately.